src/Pages/Professionals/AllProfessionals/index.js

```diff
@@ -1,13 +1,37 @@
-import React from "react"
+import React, { useState } from "react"
 import { useI18n } from "providers/I18n"
+import { usePaginatedQuery } from "react-query"
+
+import AllUsersTitle from "../../../components/AllUsersTitle"
+import ProfessionalsTable from "./ProfessionalsTable"
+import { getProfessionals } from "api/professionals"
 
 AllProfessionals.propTypes = {}
 
 function AllProfessionals() {
   const { t } = useI18n()
+
+  const [pageInfo, setPageInfo] = useState({ pageSize: 10, pageIndex: 0 })
+
+  const { isLoading, resolvedData = {} } = usePaginatedQuery(
+    ["allProfessionals", { ...pageInfo }],
```

Comment:
This is the same as this. Keep it simple!
```suggestion
    ["allProfessionals", pageInfo ],
```